dash-renderer/package.json

```diff
@@ -22,6 +23,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/polyfill": "7.8.7",
+    "@plotly/dash-component-plugins": "^1.2.0",
```

Comment:
Moved from devDeps because it is required for building the package and this way on `npm install dash-renderer` this dependency will be fetched.